scripts/updateDocs.sh

```diff
@@ -0,0 +1,42 @@
+set -eu
+
+GIT_ROOT="$(git rev-parse --show-toplevel)"
+cd $GIT_ROOT
+
+# Determine the version
+BRANCH="$(git branch | grep \* | cut -d ' ' -f2)"
+if [ "${BRANCH}" == 'develop' ]
```

Comment:
I think we only want to push documentation from:
+ develop
+ named tags (i.e. releases)
+ Master

So can you model this logic to:
- check for a tag (using `git tag -l --points-at HEAD`). If present. Great! That will be also the version number and start pushing the documentation.
- if no tag is present: check if the branch is called develop, and push to the develop documentation.
- In all other cases fail with a warning message. We only need documentation for released tags and for develop anyway.

All edge cases we can do manually.